unyt/array.py

```diff
@@ -195,7 +202,10 @@ def _square_unit(unit):
 
 @lru_cache(maxsize=128, typed=False)
 def _divide_units(unit1, unit2):
-    ret = (unit1 / unit2).simplify()
+    try:
+        ret = (unit1 / unit2).simplify()
+    except SymbolNotFoundError:
+        ret = (1 / (unit2 / unit1).simplify()).units
```

Comment:
I think there's a dimensionless `1` unit we could use here? Not sure if that would be better or not.

Reply:
I can try. How would that work ?

Reply:
This is fine.